compiler/src/dotty/tools/dotc/reporting/trace.scala

```diff
@@ -7,27 +7,39 @@ import config.Config
 import config.Printers
 import core.Mode
 
+object trace extends TraceSyntax:
+  inline def isForced: false = false
```

Comment:
I think it would be nicer to have `isEnabled` instead of `isForced`. For `trace`, 
```scala
  inline def isEnabled = Config.tracingEnabled
```
For trace.force
```scala
  inline def isEnabled = true
```

Reply:
Makes sense, done.